project-base/tests/FrontendApiBundle/Functional/Payment/PaymentsTest.php

```diff
@@ -8,6 +8,18 @@
 
 class PaymentsTest extends GraphQlTestCase
 {
+    /**
+     * @var string
+     */
+    private $overwriteDomainUrl;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->overwriteDomainUrl = $this->getContainer()->getParameter('overwrite_domain_url');
```

Comment:
This is used in each tests of this PR and I gues it will be needed in every single GQL test.

Would not be better to define this in `GraphQlTestCase`?

Reply:
Fixed